ui-tests/src/test/java/org/openmrs/reference/page/ManageAppsPage.java

```diff
@@ -0,0 +1,106 @@
+			"                \"type\": \"link\",\n" + 
+			"                \"label\": \"reportingui.reportsapp.home.title\",\n" + 
+			"                \"url\": \"reportingui/reportsapp/home.page\",\n" + 
+			"                \"icon\": \"icon-list-alt\",\n" + 
+			"                \"order\": 5,\n" + 
+			"                \"requiredPrivilege\": \"App: reportingui.reports\"\n" + 
+			"            },\n" + 
+			"        {\n" + 
+			"                \"id\": \"reportingui.dataExports.adHoc\",\n" + 
+			"                \"extensionPointId\": \"org.openmrs.module.reportingui.reports.dataexport\",\n" + 
+			"                \"type\": \"link\",\n" + 
+			"                \"label\": \"reportingui.runReport.label\",\n" + 
+			"                \"url\": \"reportingui/runReport.page\",\n" + 
+			"                \"order\": 9999,\n" + 
+			"                \"requiredPrivilege\": \"App: reportingui.runReports\",\n" + 
+			"                \"featureToggle\": \"reportingui_runReports\"\n" + 
+			"            },\n" + 
+			"            {\n" + 
+			"                \"id\": \"reportingui.dataExports.adHoc\",\n" + 
+			"                \"extensionPointId\": \"org.openmrs.module.reportingui.reports.dataexport\",\n" + 
+			"                \"type\": \"link\",\n" + 
+			"                \"label\": \"reportingui.adHocAnalysis.label\",\n" + 
+			"                \"url\": \"reportingui/adHocManage.page\",\n" + 
+			"                \"order\": 9999,\n" + 
+			"                \"requiredPrivilege\": \"App: reportingui.adHocAnalysis\",\n" + 
+			"                \"featureToggle\": \"reportingui_adHocAnalysis\"\n" + 
+			"            },\n" + 
+			"            {\n" + 
+			"                \"id\": \"Hospitalization Diagnosis Report\",\n" + 
+			"                \"extensionPointId\": \"org.openmrs.module.reportingui.reports.overview\",\n" + 
+			"                \"type\": \"link\",\n" + 
+			"                \"label\": \"Hospitalization Diagnosis Report\",\n" + 
+			"                \"url\": \"reportingui/adHocManage.page\",\n" + 
+			"                \"order\": 9999,\n" + 
+			"                \"requiredPrivilege\": \"App: reportingui.reports\"\n" + 
+			"            }\n" + 
+			"        ],\n" + 
+			"        \"requiredPrivilege\": \"App: reportingui.reports\"\n" + 
+			"    }"
+			+ "}";
+	
+	private static final By SAVE_BUTTON = By.cssSelector("#save-button");
+	
+	public ManageAppsPage(Page page) {
+		super(page);
+	}
+	
+	public void clickOnAddAppDefinition() {
+		clickOn(ADD_APP_DEFINITION);
+	}
+	
+	public void enterAppIdField(String appIdField) {
+		setTextToFieldNoEnter(APP_ID_FIELD , appIdField);
+	}
+	
+	public void enterDefinitionJsonField(String definitionjsonField) {
+		setTextToFieldNoEnter( DEFINITION_JSON_FIELD , definitionjsonField);
+	}
+	
+	public void clickOnSaveButton() {
+		clickOn(SAVE_BUTTON);
+	}
+
+	@Override
+	public String getPageUrl() {
+		return "/referenceapplication/manageApps.page";
+	}
```

Comment:
Correspond ManageAppsPage indention with the indention of existing pages within the module

Reply:
identation is the same as the whole project definition.Thanks

Reply:
@sherrif10 have you had a look at the indention of the existing pages within the module?